### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Microsoft IIS Home Page_Color Contrast: When "Download Microsoft IIS Administration" link receives keyboard focus, color contrast ratio fails for the link text. 

### ISSUE THREAD

**Issue by accessibility18** [Microsoft/IIS.WebManager#67]
**Environment Details**:
URL: https://manage.iis.net/
Browser: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106
Issue: Keyboard, Color Contrast 

**MAS ID Violated:** MAS1.4.3

**Repro Steps:**
1. Open "Microsoft IIS" Home Page using the URL: https://manage.iis.net/
2. Tab till "Download Microsoft IIS Administration" link and now verify the color contrast ratio for the link's text.

**Actual Result:**
When "Download Microsoft IIS Administration" link receives keyboard focus, color contrast ratio (2.1:1) fails for the link text. 

**Expected Result:**
Minimum color contrast ratio should be greater than or equal to 4.5:1 at any point of time. After receiving keyboard focus, color contrast should not change.

**User Impact:**
People with low vision often have difficulty reading text that does not contrast with its background. This can be exacerbated if the person has a color vision deficiency that lowers the contrast even further. Providing a minimum luminance contrast ratio between the text and its background can make the text more readable even if the person does not see the full range of colors. It also works for the rare individuals who see no color.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

**Attachment:**
![bug 67_color contrast ratio fails for download microsoft iis administration link](https://user-images.githubusercontent.com/38521133/44571091-6cb05680-a79d-11e8-8eea-3927dec1a480.png)

**Comment by jhkimnew** [comment]
Fixed

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #ColorContrast; #MAS1.4.3; #C+E-H; #.NETWebsites; #Edge

**Comment by fsteffi** [comment]
#A11ySev2;

**Comment by fsteffi** [comment]
#closed;
manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will  be removed.
